examples/oft-solana/programs/oft/src/instructions/lz_receive.rs

```diff
@@ -0,0 +1,167 @@
+use crate::*;
```

Comment:
Again, and maybe I am missing something fundamental with solana, but wouldn't we want to put the source of the oft in `packages/` not `examples/`?

Reply:
You need to deploy an OFT Program and mint accounts off of it, so you need to compile the anchor code and then deploy it directly via anchor.

After you've deployed your OFT Program, you can then mint the necessary accounts to interact with the OFT Program:

```
"accounts": [
    {
      "name": "SPL Token Mint Account",
      "publicKey": "Eh6XEPhSwoLv5wFApukmnaVSHQ6sAnoD9BmgmwQoN2sN"
    },
    {
      "name": "OFT Config Account",
      "publicKey": "D27Yp9iR2UAgD8BkBbWrHeK6MooSQu353NJm3SVmEvXw"
    },
    {
      "name": "OFT Program ID",
      "publicKey": "CKsfN2UvdF6BxWmt12576GN4EtLynA7MxSqEm6jrCqMh"
    }
  ]
```

Reply:
The same as above @ryandgoulding - I will see if cargo could install from `git` using `path`.